azure-pipelines.yml

```diff
@@ -106,19 +139,9 @@ stages:
       vmImage: $(macOSVMImage)
     
     variables:
-       - group: apple.appstore.distribution # import all variables for certificate from the library in azure devops
```

Comment:
@MatFillion I think you removed the correct indentation here. Even if I authorized the resources, still cannot found the correct variables for the build

Reply:
@agneszitte-nventive it seems the indentation wasn't the issue here. Seems like the task doesn't accept build variables. It was actually looking for $(...) as the secure file name, not the actual value of the variable. I refactored it to add the variable but it worked before so I reverted to the version where I just put the name of the file in the yml.